boards/arm/blt_plane/blt_plane.dts

```diff
@@ -0,0 +1,300 @@
+/*
+ * Copyright (c) 2025 Launch Initiative
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/f4/stm32f446Xe.dtsi>
+#include <st/f4/stm32f446r(c-e)tx-pinctrl.dtsi>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "Power Module";
+	compatible = "st,stm32f446xx";
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,console = &uart5;
+		zephyr,shell-uart = &uart5;
+
+		storage = &w25q128;
+		logfs = &lfs1;
+	};
+
+	aliases {
+		inabatt = &ina0;
+		ina3v3 = &ina1;
+		ina5v0 = &ina2;
+		vin = &adc1;
+		rtc = &rv8263;
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+
+		led_0: led_0 {
+			gpios = <&gpiob 12 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_1: led_1 {
+			gpios = <&gpiob 13 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_2: led_2 {
+			gpios = <&gpiob 14 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_3: led_3 {
+			gpios = <&gpiob 15 GPIO_ACTIVE_HIGH>;
+		};
+
+		wiznet_led: wiznet_led {
+			gpios = <&gpioc 7 GPIO_ACTIVE_HIGH>;
+		};
+
+		rtc_en_clk_out: rtc_en_clk_out {
+			gpios = <&gpiob 5 GPIO_ACTIVE_LOW>; // this defaults to enabling CLKOUT
+			label = "RV8263 Enable Clock Out";
+		};
+	};
+
+	fstab {
+		compatible = "zephyr,fstab";
+
+		lfs1: lfs1 {
+			compatible = "zephyr,fstab,littlefs";
+			mount-point = "/lfs";
+			partition = <&external_storage_partition>;
+			automount;
+
+			read-size = <256>;
+			prog-size = <256>;
+			cache-size = <512>;
+			lookahead-size = <128>;
+			block-cycles = <(90 * 1000)>;
+		};
+	};
+
+	zephyr,user {
+		io-channels = <&adc1 4>;
+	};
+};
+
+&clk_lsi {
+	status = "okay";
+};
+
+&clk_hsi {
+	clock-frequency = <DT_FREQ_M(16)>;
+	status = "okay";
+};
+
+&pll {
+	div-m = <8>;
+	mul-n = <90>;
+	div-p = <2>;
+	clocks = <&clk_hsi>;
+	status = "okay";
+};
+
+&rcc {
+	clocks = <&pll>;
+	clock-frequency = <DT_FREQ_M(45)>;
+	ahb-prescaler = <2>;
+	apb1-prescaler = <2>;
+	apb2-prescaler = <1>;
+};
+
+/*
+&adc1 {
+    pinctrl-0 = <&adc1_in4_pa4>;
+    pinctrl-names = "default";
+    st,adc-clock-source = <SYNC>;
+    st,adc-prescaler = <2>;
+    status = "okay";
+    #address-cells = <1>;
+    #size-cells = <0>;
+
+    channel@4 {
+        reg = <4>;
+        zephyr,gain = "ADC_GAIN_1";
+        zephyr,reference = "ADC_REF_INTERNAL";
+        zephyr,acquisition-time = <ADC_ACQ_TIME_DEFAULT>;
+        zephyr,resolution = <12>;
+    };
+};
+*/
+&i2c1 {
+	pinctrl-0 = <&i2c1_scl_pb6 &i2c1_sda_pb7>;
+	pinctrl-names = "default";
+	status = "okay";
+	clock-frequency = <I2C_BITRATE_FAST>;
+
+	ina0: ina219_battery@40 {
+		status = "okay";
+		compatible = "ti,ina219";
```

Comment:
are these the exact same setup and resistor vals or is jim gonna make fun of us again

Reply:
Should be the exact same setup. I also compared the current values to the power supply and they look legit.